stores/metadata.go

```diff
@@ -664,24 +664,25 @@ func (s *SQLStore) ArchiveContract(ctx context.Context, id types.FileContractID,
 }
 
 func (s *SQLStore) ArchiveContracts(ctx context.Context, toArchive map[types.FileContractID]string) error {
-	// fetch ids
-	var ids []types.FileContractID
-	for id := range toArchive {
-		ids = append(ids, id)
-	}
+	var errs []string
+	for fcid, reason := range toArchive {
+		// invalidate health of related sectors before archiving the contracts
```

Comment:
`nit` update comment to reflect we went from archiving multiple contracts to single contract